src/mailer/index.ts

```diff
@@ -2,7 +2,8 @@ import * as nodemailer from 'nodemailer'
 import Mail from 'nodemailer/lib/mailer'
 import SMTPTransport from 'nodemailer/lib/smtp-transport'
 import { readFileSync } from 'fs'
-import Mustache from 'mustache'
+import * as nunjucks from 'nunjucks'
```

Comment:
I'd like to instead pull in the named `export` for `renderString` from `nunjucks`:
```suggestion
import { renderString } from 'nunjucks'
```
We should only use `import * as module from 'module'` when it's 100% needed, otherwise pull the named export from the module.

Reply:
Could also update `nodemailer` to be:
```ts
import { createTransport, Transporter, SentMessageInfo } from 'nodemailer'
```
instead of using `import * as nodemailer from 'nodemailer'`

Reply:
Complete and Complete.